src/pages/guidelines/accessibility/color.mdx

```diff
@@ -2,12 +2,17 @@
 label: IBM firmly believes that web and software experiences should be accessible for everyone, regardless of abilities or impairments.
 title: Accessibility
 description: IBM firmly believes that web and software experiences should be accessible for everyone. Carbon is committed to following best practices when it comes to accessibility.
-tabs: ["Overview", "Color", "Keyboard", "Developers"]
+tabs: ["Overview", "Color", "Developers", "Keyboard"]
 ---
 
+<PageDescription>
+
+Visual impairments can include low vision, color blindness, and complete blindness. Carbon components are designed to accommodate the entire spectrum of visual impairment, though designers still need to exercise diligence to ensure that the components are used correctly.
+
+</PageDescription>
+
 <AnchorLinks>
 
-<AnchorLink>Introduction</AnchorLink>
 <AnchorLink>Color contrast</AnchorLink>
```

Comment:
```suggestion
<AnchorLink>Resources</AnchorLink>
```

Reply:
According to @jeanservaas our page pattern excludes the resources section as an anchorlink. I'm not opposed to changing this but it would have to be applied more broadly across the site.